tests/foreman/api/test_setting.py

```diff
@@ -0,0 +1,73 @@
+# -*- encoding: utf-8 -*-
+"""Test class for Setting Parameter values
+
+:Requirement: Setting
+
+:CaseAutomation: Automated
+
+:CaseLevel: Acceptance
+
+:CaseComponent: API
+
+:TestType: Functional
+
+:CaseImportance: High
+
+:Upstream: No
+"""
+from robottelo.decorators import tier1, stubbed
+from robottelo.test import APITestCase
+
+
+class SettingTestCase(APITestCase):
+    """Implements tests for Settings for API"""
+
+    @stubbed
+    @tier1
+    def test_positive_update_login_page_footer_text(self):
+
+        """Updates parameter "login_text" in settings
+
+        :id: 91c5373d-b928-419d-8509-761adf5b94b0
+
+        :steps:
+
+            1. Create login text with any string
+
+        :expectedresults: Parameter is updated successfully
+
+        :caseautomation: notautomated
+        """
+
+    @stubbed
+    @tier1
+    def test_positive_update_login_page_footer_text_without_value(self):
+        """Updates parameter "login_text" without any string (empty value)
+
+        :id: 7a56f194-8bde-4dbf-9993-62eb6ab10733
+
+        :steps:
+
+            1. Create login text without any string(empty value)
+
+        :expectedresults: Message on login screen should be removed
```

Comment:
@oshtaier  no need to verify that from UI...read that parameter from API and it should be empty.